core/tests/protractor_utils/BlogDashboardPage.js

```diff
@@ -0,0 +1,368 @@
+  var blogPostContentEditor = element(
+    by.css('.protractor-test-content-editor'));
+  var blogPostTitleFieldElement = element(
+    by.css('.protractor-test-blog-post-title-field'));
+  var listViewButton = element(
+    by.css('.protractor-test-list-view-button'));
+  var tilesViewButton = element(
+    by.css('.protractor-test-tiles-view-button'));
+  var draftBlogPostsTable = element(
+    by.css('.protractor-test-drafts-blog-post-table'));
+  var publishedBlogPostsTable = element(
+    by.css('.protractor-test-published-blog-post-table'));
+  var blogPostListItems = element.all(
+    by.css('.protractor-test-blog-post-list-item'));
+  var draftBlogPostTiles = element.all(
+    by.css('.protractor-test-draft-blog-post-tile-item'));
+  var publishedBlogPostTiles = element.all(
+    by.css('.protractor-test-published-blog-post-tile-item'));
+  var blogPostTags = element.all(
+    by.css('.protractor-test-blog-post-tags'));
+  var saveBlogPostAsDraftButton = element(
+    by.css('.protractor-test-save-as-draft-button'));
+  var publishBlogPostButton = element(
+    by.css('.protractor-test-publish-blog-post-button'));
+  var thumbnailClickable = element(
+    by.css('.protractor-test-photo-clickable'));
+  var customThumbnail = element(
+    by.css('.protractor-test-custom-photo'));
+  var thumbnailCropper = element(
+    by.css('.protractor-test-photo-crop .cropper-container'));
+  var thumbnailUploadError = element(
+    by.css('.protractor-test-upload-error'));
+  var saveBlogPostContentButton = element(
+    by.css('.protractor-test-save-blog-post-content'));
+  var blogPostContentDisplay = element(
+    by.css('.protractor-test-content-display'));
+  var confirmButton = element(
+    by.css('.protractor-test-confirm-button'));
+  var currUsername = element(
+    by.css('.protractor-test-username-visible'));
+  var cancelThumbnailUploadButton = element(
+    by.css('.protractor-test-photo-upload-cancel'));
+  var blogDashboardIntroMessageContainer = element(
+    by.css('.protractor-test-intro-message-container'));
+  var closeBlogCardPreviewButton = element(
+    by.css('.protractor-test-close-preview-button'));
+  var blogCardPreviewButton = element(
+    by.css('.protractor-test-blog-card-preview-button'));
+  var blogDashboardLink = element(by.css(
+    '.protractor-test-blog-dashboard-link'));
+  var blogPostTiles = element.all(by.css(
+    '.protractor-test-blog-dashboard-tile'));
+  var blogPostTileTitle = by.css(
+    '.protractor-test-blog-post-title');
+  var navigateToBlogDashboardButton = element(
+    by.css('.protractor-test-back-button'));
+
+  this.get = async function() {
+    await waitFor.pageToFullyLoad();
+    await general.openProfileDropdown();
+    await action.click(
+      'Blog dashboard link from dropdown', blogDashboardLink);
+    await waitFor.pageToFullyLoad();
+    await waitFor.urlRedirection('http://localhost:9001/blog-dashboard');
+  };
+
+  this.navigateToBlogDashboardPage = async function() {
```

Comment:
Maybe `navigateToBlogDashboardPageWithBackButton` to make clear how this function differs from `get()`?

Reply:
Done